latch_cli/main.py

```diff
@@ -201,20 +209,25 @@ def dockerfile(pkg_root: str, snakemake: bool = False):
     crash_handler.message = "Failed to generate Dockerfile."
     crash_handler.pkg_root = pkg_root
 
-    from latch_cli.docker_utils import generate_dockerfile
+    if snakemake is True and nextflow is True:
+        click.secho(
+            f"Please specify only one workflow type to generate metadata for. Use"
+            f" either `--snakemake` or `--nextflow`.",
+            fg="red",
+        )
+        raise click.exceptions.Exit(1)
+
+    from latch_cli.docker_utils import generate_dockerfile, generate_dockerignore
 
-    source = Path(pkg_root)
-    dest = source / "Dockerfile"
-    if dest.exists() and not click.confirm(
-        f"Dockerfile already exists at `{dest}`. Overwrite?"
-    ):
-        return
     workflow_type = WorkflowType.latchbiosdk
     if snakemake is True:
         workflow_type = WorkflowType.snakemake
-    generate_dockerfile(source, dest, wf_type=workflow_type)
+    elif nextflow is True:
+        workflow_type = WorkflowType.nextflow
 
-    click.secho(f"Successfully generated dockerfile `{dest}`", fg="green")
+    source = Path(pkg_root)
+    generate_dockerfile(source, wf_type=workflow_type)
+    generate_dockerignore(source, wf_type=workflow_type)
```

Comment:
gate this behind a `click.confirm` -

Reply:
Its already gated within the function to prevent overwrite